tools/cloud_functions/gcs_event_based_ingest/gcs_ocn_bq_ingest/main.py

```diff
@@ -376,7 +389,7 @@ def get_batches_for_prefix(storage_client,
     (one batch has an array of multiple GCS uris)
     """
     batches = []
-    bucket_name, prefix_name = _parse_gcs_url(prefix_path)
+    bucket_name, prefix_name = parse_gcs_url(prefix_path)
```

Comment:
I'd recommend using as much of the cloud storage python client as possible. You can get rid of your parsing function and use:
https://googleapis.dev/python/storage/latest/blobs.html#google.cloud.storage.blob.Blob.from_string

Reply:
TIL this existed thanks <3